### ISSUE TITLE
Option for locking and quick unlocking encrypted databases

### ISSUE THREAD

**Issue by BanCrash** [moneymanagerex/moneymanagerex#5150]
# Description of the feature
Hi, I'm using an encrypted database and I found it a little annoying that I have to close and open each time I want to use it because I don't want to have it unlocked all the time, just when I want to use it. Other programs, like Keepass for example, have an option that I find very useful:

1st time opening the database: You have to enter the password as usual.
When you ended doing anything: Instead of closing the program, just lock the database. So, when you want to use it again, you have to just unlock it. But if you want, you don't have to enter the full password the second and after times, you can just use Windows Hello / Touch Id, which is faster. I understand that this is a multiplatform program and so maybe it's not worth it to add support for those systems, but it can be simplified to a PIN that you can set on the settings. This would be optional of course.

So, in summary:
Open the database -> Lock the database instead closing the program when you are not going to use for a long time MMex -> Unlock the database when you want to use it again, using an already set PIN or entering the database password as usual. 

# Use cases
I'm closing and opening the program from time to time because I don't want to have it unlocked when I think I'm not going to use it for a long time. And sometimes I forgot to do something and I have to reopen the program and enter the full password again, while in Keepass for example I only have to quickunlock the database since the program is already open.

**Comment by BanCrash** [comment]
> This issue is stale because it has been open 365 days with no activity. Please update if you want to keep the issue open

I still want this.

**Comment by BanCrash** [comment]
I still want this feature
